### DISCUSSION TITLE
Highres fix UI change

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#1181] [Ideas]
I noticed an UI change when activating Highres fix.
Denoising strength and Upscale by has been swapped, which breaks the workflow IMHO. Upscale by should be underneath the upscaler and the Denoising strength underneath the Highres steps so that related elements are arranged logical and among each other.

**Comment by vladmandic** [comment]
fixed.

**Comment by Woisek** [comment]
Hmm ... not how I meant, but it's OK, too I guess ... 😁
Thanks.
